apps/hash-frontend/src/pages/types/types-page-tabs.tsx

```diff
@@ -0,0 +1,83 @@
+import { Tabs, tabsClasses, typographyClasses } from "@mui/material";
```

Comment:
Strictly speaking this file and `types-table.tsx` (and therefore the `types-table` folder) should be in the `[[...type-kind]].page` folder, because they are its private dependencies

Reply:
Ah yes, fair enough. If I'm understanding right this also applies to the other components defined in that directory, have moved them in https://github.com/hashintel/hash/pull/2908/commits/116df25ed6a56f457e8c53368ae844dffcf8de9b